ngraph_bridge/grappler/ngraph_optimizer.cc

```diff
@@ -230,7 +262,8 @@ Status NgraphOptimizer::Optimize(tensorflow::grappler::Cluster* cluster,
 
   // 4. Encapsulate clusters then, if requested, dump the graphs.
   FunctionDefLibrary* fdeflib_new = new FunctionDefLibrary();
-  TF_RETURN_IF_ERROR(EncapsulateClusters(&graph, idx, fdeflib_new, config_map));
+  TF_RETURN_IF_ERROR(
+      EncapsulateClusters(&graph, idx, fdeflib_new, config_map, aot_info));
```

Comment:
TODO: what if we call "AOT+shapehints" from sess.run? (instead of through tf2ngraph